Anchor.toml

```diff
@@ -27,4 +27,4 @@ cluster = "localnet"
 wallet = "~/.config/solana/id.json"
 
 [scripts]
-test = "yarn run ts-mocha -t 1000000 tests/gummyroll.ts"
+test = "yarn run ts-mocha -t 1000000 tests/**/*-test.ts"
```

Comment:
This change makes the `gummyroll_crud` tests (et al.) run again, which will catch cross-program breakages. Also, added a `-test` suffix so that mocha doesn't try to run the non-tests in the `tests/` folder.